source/sass/components/take-action.scss

```diff
@@ -0,0 +1,17 @@
+.take-action-section {
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.take-action-title {
+  font-family: $font-family-sans-serif;
+  font-size: 23px;
+  font-weight: bold;
+  line-height: 1.3;
+  color: $black;
+  margin-bottom: 27px;
```

Comment:
- Replace rules for font-size and font-height with `@include set-text-size(23px, 27px)`. 
- font color should already be black so `color: $black;` is not needed
- Bottom margin CSS rule can be replaced by applying a `mb-4` class to `take-action-title`